conference/src/test/java/com/kyriba/conference/management/api/HallApiTest.java

```diff
@@ -49,6 +57,9 @@
   @LocalServerPort
   int port;
 
+  @MockBean
+  HallService hallService;
```

Comment:
Don't use mocks in API tests, run H2 database or real DB with testcontainers

Reply:
done